cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlannerTest.java

```diff
@@ -118,7 +118,7 @@ public void testGetPartitionMovementTasks() {
     proposals.add(partitionMovement4);
     // Test different execution strategies.
     ExecutionTaskPlanner basePlanner = new ExecutionTaskPlanner(null);
-    ExecutionTaskPlanner postponeUrpPlanner = new ExecutionTaskPlanner(Collections.singletonList(PostponeUrpReplicaMovementStrategy.class.getName()));
+    ExecutionTaskPlanner postponeUrpPlanner = new ExecutionTaskPlanner(null);
```

Comment:
IIUC, we made this change to test dynamic input rather than passing a value in creation time? If this is the case, can we perform this test in another test case, and keep this one the same as before?